src/Sylius/Bundle/AdminBundle/Resources/config/app/twig_hooks/product/show.yaml

```diff
@@ -51,3 +37,94 @@ twig_hooks:
                 template: '@SyliusAdmin/product/show/content/page_body/taxonomy/main_taxon.html.twig'
             product_taxons:
                 template: '@SyliusAdmin/product/show/content/page_body/taxonomy/product_taxons.html.twig'
+
+        'sylius_admin.product.show.content.page_body.details':
+            simple_product:
+                template: '@SyliusAdmin/product/show/content/page_body/details/simple_product.html.twig'
+            configurable_product:
+                template: '@SyliusAdmin/product/show/content/page_body/details/configurable_product.html.twig'
+
+        'sylius_admin.product.show.content.page_body.details.simple_product':
+            pricing:
+                template: '@SyliusAdmin/product/show/content/page_body/details/simple_product/pricing.html.twig'
+            shipping:
+                template: '@SyliusAdmin/product/show/content/page_body/details/shared/shipping.html.twig'
+                
+                
+        'sylius_admin.product.show.content.page_body.details.simple_product.shipping':
+            shipping_category:
+                template: '@SyliusAdmin/product/show/content/page_body/details/shared/shipping/shipping_category.html.twig'
+            width:
+                template: '@SyliusAdmin/product/show/content/page_body/details/shared/shipping/width.html.twig'
+            height:
+                template: '@SyliusAdmin/product/show/content/page_body/details/shared/shipping/height.html.twig'
+            depth:
+                template: '@SyliusAdmin/product/show/content/page_body/details/shared/shipping/depth.html.twig'
+            weight:
+                template: '@SyliusAdmin/product/show/content/page_body/details/shared/shipping/weight.html.twig'
```

Comment:
Just a thought, but can we make a `details.common` hook and set it as one of the prefixes for data that's the same between configurable/simple?
Right now there's a lot of duplication.
Unless the prefixing logic is not yet released, in which case, welp.